edx_django_utils/hooks/tasks.py

```diff
@@ -3,12 +3,15 @@
 """
 from logging import getLogger
 
+from celery import shared_task
+
 from .exceptions import HookException
 from .utils import get_functions_for_pipeline
 
 log = getLogger(__name__)
 
 
+@shared_task()
```

Comment:
Something interesting that I found in edx-celeryutils was a BaseTask called [LoggedTask](https://github.com/edx/edx-celeryutils/blob/be8c68a72d144d720f28622f1963330c451b20f6/celery_utils/logged_task.py#L13), wouldn't be useful to have that extra logging?